bin/shpec

```diff
@@ -121,61 +121,62 @@ assert() {
 
 print_result() {
   if eval "$1"; then
-    iecho "$green$assertion$norm"
+    iecho "$_shpec_green$_shpec_assertion$_shpec_norm"
   else
-    : $((failures += 1))
-    iecho "$red$assertion"
-    iecho "($2)$norm"
+    : $((_shpec_failures += 1))
+    iecho "$_shpec_red$_shpec_assertion"
+    iecho "($2)$_shpec_norm"
   fi
 }
 
 final_results() {
-  [ $failures -eq 0 ] && color=$green || color=$red
-  echoe "${color}${examples} examples, ${failures} failures${norm}"
+  [ $_shpec_failures -eq 0 ] && _shpec_color=$_shpec_green || _shpec_color=$_shpec_red
+  echoe "${_shpec_color}${_shpec_examples} examples, ${_shpec_failures} failures${_shpec_norm}"
   times
-  [ $failures -eq 0 ]
+  [ $_shpec_failures -eq 0 ]
   exit
 }
 
 
 shpec() {
   (
-    VERSION=0.2.1
+    _shpec_VERSION=0.2.1
```

Comment:
Ah, I think I found an elegant way out.

    shpec_version() {
      (
      VERSION=0.2.1
      echo $VERSION
      )
    }
    
    shpec() {
      (
        case "$1" in
        ( -v | --version )
          shpec_version
          exit 0
        ;;
        esac